internal/kafka/internal/presenters/kafka.go

```diff
@@ -86,3 +88,14 @@ func setBootstrapServerHost(bootstrapServerHost string) string {
 	}
 	return bootstrapServerHost
 }
+
+func getDisplayName(instanceType string) string {
+	switch instanceType {
+	case types.DEVELOPER.String():
+		return types.DEVELOPER_DISPLAY_NAME.String()
+	case types.STANDARD.String():
+		return types.STANDARD_DISPLAY_NAME.String()
+	default:
+		return ""
+	}
+}
```

Comment:
I believe this method should be part of `KafkaInstanceType`

Reply:
this function is now removed (values are obtained from the kafkaConfig)